### DISCUSSION TITLE
USD 21.11 own build - what patches are needed?

### DISCUSSION THREAD

**Discussion by DanEnglesson** [Autodesk/maya-usd#3457] [Q&A] [answered]
Hi,

Could you guys provide which patches you apply to USD 21.11 for Maya 2022 to keep instancing segfaults at bay?
We, for various reason, build our own USD against Maya but it is segfaulting with any instancing in the scene. This was fixed in 2023 with the introduction of USD 22.11 but you backported patches to make it work for 21.11.

I couldn't find any patch information that you are running. Any info on this would greatly help as we cannot yet move to a newer version of Maya.

Many thanks,
Dan

**Comment by seando-adsk** [comment]
Hi Dan, here is a list of commits that we have cherry picked in our build of USD v21.11.

https://github.com/PixarAnimationStudios/OpenUSD/commit/08a5644770d063fe355aff1a2069b22b84d8402a
https://github.com/PixarAnimationStudios/OpenUSD/commit/af6eebc44a72a5da646d28c10cac2622e5bb07ed
https://github.com/PixarAnimationStudios/OpenUSD/commit/13d99dd533964b72149e8e446df1bb6d428b9640
https://github.com/PixarAnimationStudios/OpenUSD/commit/6f5b22e1d7fbe27ea6535afe08970bda089bfd27
https://github.com/PixarAnimationStudios/OpenUSD/commit/26474137e75008709e92f1b1243173d1cdfcd683
https://github.com/PixarAnimationStudios/OpenUSD/commit/6f1668ff1683bded82c8faefa3281f25fe1bcfeb
https://github.com/PixarAnimationStudios/OpenUSD/commit/d5ac7f9dd800a74db01d85b32828ecc98c1aa20c
https://github.com/PixarAnimationStudios/OpenUSD/commit/f7f976ce2b42595957deae2418d2ae6c9f90b54b

We are looking into publishing our build commits.

Sean

**Reply by DanEnglesson** [reply]
Many thanks Sean!

**Comment by seando-adsk** [comment, marked as answer]
The source code to build the various USD versions has been pushed to the Autodesk Github USD fork (with tags). Please see the column "USD source for MayaUsd" in the table in section [Download and Build Pixar USD](https://github.com/Autodesk/maya-usd/blob/dev/doc/build.md#2-download-and-build-pixar-usd)

Sean
